package.json

```diff
@@ -195,6 +195,7 @@
     "sleep-promise": "^8.0.1",
     "stream-browserify": "^3.0.0",
     "svgs": "^4.1.1",
+    "ts-results": "^3.3.0",
```

Comment:
do you experience the dependency issue for ts-results Joe experienced when importing the sdk?